app/components/ui/Navigation.module.scss

```diff
@@ -172,3 +178,29 @@
     font-style: normal;
   }
 }
+
+.buttonLink {
+  appearance: button;
+  text-decoration: none;
+  color: white;
+  background-color: $color-brand;
+  border-radius: 3px;
+  font-size: 16px;
+  font-weight: 600;
+  height: 60px;
+  width: 200px;
```

Comment:
I generally avoid setting both the height and width of elements, since that can cause undesirable behavior when the element's content is larger than height and/or width. Even if it looks fine right now, we might change the text later, or it may look wrong on a different browser/OS that has a different text rendering algorithm.

For buttons, I normally like controlling the height via padding, not height, which allows it to look natural in the case where you decide to allow the text in the button to wrap to multiple lines. I also usually like primarily controlling the width via padding as well, although sometimes it's nice to set a max and/or min width.